### DISCUSSION TITLE
Approve bot only works sometime

### DISCUSSION THREAD

**Discussion by vicary** [renovatebot/renovate#18993] [Request Help] [answered]
We added the approve bot in our repository and it worked for some of the PRs (e.g. https://github.com/gqty-dev/gqty/pull/1265), while others (e.g. https://github.com/gqty-dev/gqty/pull/1267) doesn't receive an approve.

How should I troubleshoot this?

**Comment by rarkins** [comment, marked as answer]
There is no bug. The approve bot only approves automerging PRs. The one you link to doesn't have automerge enabled.

**Reply by vicary** [reply]
Yes I mixed it up with other non-merging PRs, will update if it happens again. Thanks for the answer.
